go.mod

```diff
@@ -206,17 +207,23 @@ replace github.com/openshift/api => github.com/openshift/api v0.0.0-202108310919
 // Required for the marin3r which uses an older version of the operator-lib library - https://issues.redhat.com/browse/MGDAPI-4893
 replace github.com/operator-framework/operator-lib => github.com/operator-framework/operator-lib v0.1.0
 
+replace k8s.io/apimachinery => k8s.io/apimachinery v0.24.3
+
+replace k8s.io/client-go => k8s.io/client-go v0.24.3
+
+replace k8s.io/component-base => k8s.io/component-base v0.24.3
+
+replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.24.3
+
+replace k8s.io/apiserver => k8s.io/apiserver v0.24.3
```

Comment:
I know this is a wip and would be subject to change when the correct OO version is being used. But we agree that any `replace` needs to be commented to why its being done.

Reply:
Agreed. I mentioned it in my comment above. I believe marin3r is now the offender given that OO is moved out to submodule. Can leave a comment in the code for that for sure.